dds/DCPS/RTPS/Sedp.cpp

```diff
@@ -4227,8 +4227,10 @@ Sedp::Reader::data_received(const DCPS::ReceivedDataSample& sample)
     Serializer ser(payload.get(), encoding);
     DCPS::EncapsulationHeader encap;
     if (!(ser >> encap)) {
-      ACE_ERROR((LM_ERROR, ACE_TEXT("(%P|%t) ERROR: Sedp::Reader::data_received - ")
-        ACE_TEXT("failed to deserialize encapsulation header\n")));
+      if (log_level >= LogLevel::Warning) {
+        ACE_ERROR((LM_WARNING, "(%P|%t) WARNING: Sedp::Reader::data_received - "
+                   "failed to deserialize encapsulation header\n"));
+      }
```

Comment:
I don't know if this is in the guidelines but I have been prefix
```suggestion
      if (log_level >= LogLevel::Warning) {
        ACE_ERROR((LM_WARNING, "(%P|%t) WARNING: Sedp::Reader::data_received: "
                   "failed to deserialize encapsulation header\n"));
      }
```

Reply:
and so on.

Reply:
> I don't know if this is in the guidelines but I have been prefix

It is in the guidelines